src/config/config.local.json

```diff
@@ -0,0 +1,4 @@
+{
+  "ethrDid": "",
```

Comment:
The nice dev env flow would:
1. Start ganache
2. Deploy DID Registry on ganache
3. Re-write this file setting the address

Let's open an issue for this

Reply:
Yes, perhaps rewrite a new file so it doesn't change the developer's settings. Ticket created here: https://github.com/rsksmart/rif-identity-manager/issues/7